app/Filament/Pages/Settings.php

```diff
@@ -0,0 +1,342 @@
+                                ->default(env('MAIL_ENCRYPTION', config('mail.mailers.smtp.encryption', 'tls'))),
+                            TextInput::make('MAILGUN_DOMAIN')
+                                ->label('Mailgun Domain')
+                                ->required(true)
+                                ->visible(fn (Get $get) => $get('MAIL_MAILER') === 'mailgun')
+                                ->default(env('MAILGUN_DOMAIN', config('services.mailgun.domain'))),
+                            TextInput::make('MAILGUN_SECRET')
+                                ->label('Mailgun Secret')
+                                ->required(true)
+                                ->visible(fn (Get $get) => $get('MAIL_MAILER') === 'mailgun')
+                                ->default(env('MAIL_USERNAME', config('services.mailgun.secret'))),
+                            TextInput::make('MAILGUN_ENDPOINT')
+                                ->label('Mailgun Endpoint')
+                                ->required(true)
+                                ->visible(fn (Get $get) => $get('MAIL_MAILER') === 'mailgun')
+                                ->default(env('MAILGUN_ENDPOINT', config('services.mailgun.endpoint'))),
+                        ]),
+                    Tab::make('misc')
+                        ->label('Misc')
+                        ->icon('tabler-tool')
+                        ->schema([
+                            Section::make('Automatic Allocation Creation')
+                                ->description('Toggle if Users can create allocations via the client area.')
+                                ->columns(2)
+                                ->schema([
+                                    Toggle::make('PANEL_CLIENT_ALLOCATIONS_ENABLED')
+                                        ->label('Status')
+                                        ->onIcon('tabler-check')
+                                        ->offIcon('tabler-x')
+                                        ->onColor('success')
+                                        ->offColor('danger')
+                                        ->live()
+                                        ->columnSpanFull()
+                                        ->formatStateUsing(fn ($state): bool => (bool) $state)
+                                        ->afterStateUpdated(fn ($state, Set $set) => $set('PANEL_CLIENT_ALLOCATIONS_ENABLED', (bool) $state))
+                                        ->default(env('PANEL_CLIENT_ALLOCATIONS_ENABLED', config('panel.client_features.allocations.enabled'))),
+                                    TextInput::make('PANEL_CLIENT_ALLOCATIONS_RANGE_START')
+                                        ->label('Starting Port')
+                                        ->required(true)
+                                        ->numeric()
+                                        ->minValue(1024)
```

Comment:
i think this has to be 1025, as the ports have to be > 1024? i might be wrong

Reply:
No, ports below 1024 are restricted. So 1 - 1023 isn't allowed.

Reply:
Correct. Greater than 1024. So the min would be 1025?

Reply:
Below 1024 is 1-1023. The usable range is 1024 and above.

Reply:
Ports in an allocation must be greater than 1024 and less than or equal to 65535.
![image](https://github.com/user-attachments/assets/63160520-3e16-47e3-a640-9ce5f2c55ad0)

Reply:
> Ports in an allocation must be greater than 1024 and less than or equal to 65535. ![image](https://private-user-images.githubusercontent.com/40749467/349213332-63160520-3e16-47e3-a640-9ce5f2c55ad0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjExNTExMjQsIm5iZiI6MTcyMTE1MDgyNCwicGF0aCI6Ii80MDc0OTQ2Ny8zNDkyMTMzMzItNjMxNjA1MjAtM2UxNi00N2UzLWE2NDAtOWNlNWYyYzU1YWQwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE2VDE3MjcwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIzNTE3NGUwYzUyMGMwMDc1MDA5MTVjZDQ4YjNkYjY5OWY0YTYyZGM5MDQxZjk0MzYwNmZlYWY0ZDc3YmFlNWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Mtiyt0QKbbgb7s6FxJNs_E_xSiFtpmdwstJOvzWS-2Y)

Then we need to update that bit. It's definitely _below_ 1024, so 1024 itself is valid.